torch/csrc/lazy/core/tensor_impl.cpp

```diff
@@ -187,8 +187,7 @@ bool LTCTensorImpl::is_contiguous(c10::MemoryFormat _unused) const {
 }
 
 const at::Storage& LTCTensorImpl::storage() const {
-  TORCH_CHECK("Lazy tensors do not have storage");
-  return storage_;
```

Comment:
I think we might need to tell clang-tidy about not complaining there is no return statement here. Or you could just leave the return statement intact. The `TORCH_CHECK` should crash the program already.

Reply:
it looks like clang-tidy is happy without a return statement. i'm not sure if it's required if there is an assert?

Reply:
Or my memory could have a glitch. I remembered I added the return statement because of a warning.